### ISSUE TITLE
Ignore all rules per line comment

### ISSUE THREAD

**Issue by marcellodesales** [hadolint/hadolint#280]
- [ ] This is a bug report
- [x] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

* I'd like to ignore all rules per line

```Dockerfile
# hadolint ignore=*
ONBUILD ENV RUNNER_CMD_EXEC=${RUNNER_CMD_EXEC:-"java \$JAVA_OPTS -jar /runtime/server.jar \$JAR_OPTS"}
```

I'm getting errors and I'd like to just ignore them per line

### Actual behavior

* I can't disable syntax errors

```
./runner/openjdk-8/Dockerfile:87:54 unexpected '\' expecting '"', ''', a string with no quotes, or at least one space
```

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
$ hadolint --version
Haskell Dockerfile Linter 1.15.0-no-git
```

**Dockerfile** (if relevant)

```Dockerfile
ONBUILD ENV RUNNER_CMD_EXEC=${RUNNER_CMD_EXEC:-"java \$JAVA_OPTS -jar /runtime/server.jar \$JAR_OPTS"}
```

**Additional environment details (OS, stack version, etc.)**

* MacOS, Docker version too

**Comment by lorenzo** [comment]
This style of env vars is currently not supported `RUNNER_CMD_EXEC=${RUNNER_CMD_EXEC:-...`

The actual issue should be about supporting this style instead of the ability to disable everything.

**Comment by lorenzo** [comment]
I'll close this in favour of the similar issues https://github.com/hadolint/hadolint/issues/279

Thanks for reporting!

**Comment by marcellodesales** [comment]
@lorenzo thank u! Is there a workaround? I needed to exclude it from my checks :(

**Comment by marcellodesales** [comment]
@lorenzo  What's  a syntax that would work, in this case? I have  tried different things :( NO luck :(

**Comment by lorenzo** [comment]
Whew can RUNNER_CMD_EXEC be defined for the first time?

**Comment by henri9813** [comment]
Hello,

How can i ignore ONE RULE for ONLY one line ?

Best regards

**Comment by lorenzo** [comment]
@henri9813 yes! please check the documentation in our README

**Comment by henri9813** [comment]
Oh sorry, i just found it now, i read the doc to quickly.

Thank you !
